packages/api/src/controllers/ValidatorsController.js

```diff
@@ -20,7 +20,9 @@ class ValidatorsController {
 
     const validators = await TenderdashRPC.getValidators()
 
-    const proTxInfo = await DashCoreRPC.getProTxInfo(validator.proTxHash)
+    const transaction = await DashCoreRPC.getRawTransaction(validator.proTxHash)
```

Comment:
You don't need to get transactions for each validator. Do it only in case it is not found, there is a specific error code for that. Do it in the dashcoreRpc.js